samples/Exporters/Console/TestConsoleActivity.cs

```diff
@@ -0,0 +1,47 @@
+﻿// <auto-generated/>
+
+using OpenTelemetry.Trace.Configuration;
+using OpenTelemetry.Exporter.Console;
+using OpenTelemetry.Trace;
+using System.Diagnostics;
+using OpenTelemetry.Trace.Export;
+
+namespace Samples
+{
+    internal class TestConsoleActvity
+    {
+        internal static object Run(ConsoleActivityOptions options)
+        {
+            // Setup exporter
+            var exporterOptions = new ConsoleExporterOptions
+            {
+                Pretty = options.Pretty
+            };
+            var activityExporter = new ConsoleActivityExporter(exporterOptions);
+
+            // Setup processor
+            var activityProcessor = new SimpleActivityProcessor(activityExporter);
+
+            // Enable OpenTelemetry for the "source" named "DemoSource".
+            OpenTelemetrySDK.EnableOpenTelemetry("DemoSource", activityProcessor);
+
+            // Everything above this line is required only in Applications
+            // which decide to use OT.
+
+            // The following is generating activity.
+            // A library would simply write the following line of code.
+            var source = new ActivitySource("DemoSource");
```

Comment:
`DemoSource` is expected to be namespaced? I understand this is an equivalent of a tracer name, correct? Can you use it as intended, please?